### ISSUE TITLE
image

### ISSUE THREAD

**Issue by chaous** [123jimin/ComfyUI-MobileForm#1]
Is it possible to upload image here somehow and crate mask for it? without using whole compyUI

**Comment by 123jimin** [comment]
Nope. currently this extension only supports text / number / combo inputs.
However, I'm trying to make it work with image output nodes, and I'll try adding supports for image input forms. (Can't make it a promise, though.)
